article/templates/article/landing_page.html

```diff
@@ -47,6 +48,13 @@ <h1 class="heading-xlarge heading-hero-domestic title" id="hero-campaign-section
   </div>
 </section>
 
+{% if features.LANDING_PAGE_EU_EXIT_BANNER_ON %}
+  <br>
```

Comment:
ideally shouldn't use `br` tags for spacing unless it's in a block of text. if you need more space at the top of the banner you can add a class like `padding-top-30` or `margin-top-30`